core/xchain/entitlement/check_operation.go

```diff
@@ -179,11 +179,14 @@ func (e *Evaluator) evaluateMockOperation(
 	if result != nil {
 		return false, result
 	}
-	if op.ChainID.Sign() != 0 {
-		return true, nil
-	} else {
-		return false, nil
+
+	if (op.ContractAddress != common.Address{}) {
+		// Grab last two digits of contract address as a unique identifier of which check
+		// caused the error, for ease of debugging test cases.
+		code := fmt.Sprintf("%v", op.ContractAddress)[40:]
+		return false, fmt.Errorf("intentional failure (%v)", code)
```

Comment:
You can also do `fmt.Errorf("intentional failure (%x%x)", op.ContractAddress[18], op.ContractAddress[19])` to save on mem allocations.